data/json/npcs/talk_tags.json

```diff
@@ -259,7 +313,12 @@
       "I'm done for",
       "I won't last much longer",
       "my days are <really> numbered",
-      "<name_g>, I'm sorry"
+      "<name_g>, I'm sorry",
+      "Didn't think it would end like this.",
```

Comment:
```suggestion
      "I didn't think it would end like this",
```

Reply:
The "I" is not necessary. While technically gramatically incorrect, to start with a verb when referring to one self is a very common slang.